cmd/osbuild-image-tests/main_test.go

```diff
@@ -299,6 +299,12 @@ func testBoot(t *testing.T, imagePath string, bootType string, outputID string)
 	}
 }
 
+func kvmAvailable() bool {
+	cmd := exec.Command("modprobe", "kvm")
```

Comment:
Any reason you are not just checking for `/dev/kvm`?

Reply:
No, I changed to `/dev/kvm` detection.